<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceDiagramWindow</class>
 <widget class="QMainWindow" name="SequenceDiagramWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Classes</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="classesView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Instances</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="instancesView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Methods</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="methodView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Functions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openFileButton">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveFileButton">
          <property name="text">
           <string>Save File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createInstanceButton">
          <property name="text">
           <string>Create Instance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteInstanceButton">
          <property name="text">
           <string>Delete Instance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="visibilityButton">
          <property name="text">
           <string>Instance Visibility</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prolongueButton">
          <property name="text">
           <string>Prolongue line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shortenButton">
          <property name="text">
           <string>Shorten line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addInitBlockButton">
          <property name="text">
           <string>Add initial block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prolongueBlockButton">
          <property name="text">
           <string>Prolongue block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shortenBlockButton">
          <property name="text">
           <string>Shorten block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="syncAsyncButton">
          <property name="text">
           <string>Sync/Async</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="returnMessageButton">
          <property name="text">
           <string>Return Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setMethodButton">
          <property name="text">
           <string>Set Method</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
